### DISCUSSION TITLE
German translation

### DISCUSSION THREAD

**Discussion by flow-motion** [akeeba/panopticon#857] [Ideas]
Hey there 
I mentioned some translation probs. I would be here to help. 
E.g. in Extension Update list the button says: Chosen update in German :) 
Not a problem - but it would be fine to change this... 

I guess it should be in the end: Install updates :) 

Whenever you need some help - let me know. Have a great weekend and thanks for your efforts for the community.

**Comment by nikosdion** [comment]
There's currently no maintainer for the German translation. Until someone expresses interest to take it over, I had to remove it from the distribution.

**Comment by flow-motion** [comment]
May be we can find a team of 3 or 4 guys to do it. I would be in : )
